sandbox/namespace_linux.go

```diff
@@ -55,9 +55,7 @@ func createBasePath() {
 }
 
 func removeUnusedPaths() {
-	for {
-		time.Sleep(time.Duration(gpmCleanupPeriod))
-
+	for range time.Tick(time.Duration(gpmCleanupPeriod)) {
```

Comment:
it's needed because `gpmCleanupPeriod` is declared as 

` var gpmCleanupPeriod = 60 * time.Second`

 (int) instead of

` var gpmCleanupPeriod time.Duration = 60 * time.Second`